SUMMARY.md

```diff
@@ -2,81 +2,107 @@
+  * [Pushing assets to the browser](tutorials/core-concepts/07-front-end-assets/pushing-assets.md)
+  * [Lean frontend assets: Apostrophe without jQuery](tutorials/core-concepts/07-front-end-assets/lean-frontend-assets.md)
+* [Users and Permissions](tutorials/core-concepts/08-users-and-permissions/README.md)
+  * [Users and Groups](tutorials/core-concepts/08-users-and-permissions/users-and-groups.md)
+  * [Managing Access Control](tutorials/core-concepts/08-users-and-permissions/managing-access-control.md)
+  * [User Login and Password Help](tutorials/core-concepts/08-users-and-permissions/user-login-and-password.md)
+* [Working With Templates](tutorials/core-concepts/09-working-with-templates/README.md)
+  * [Nunjucks Filters](tutorials/core-concepts/09-working-with-templates/nunjucks-filters.md)
+  * [Nunjucks Helper Functions](tutorials/core-concepts/09-working-with-templates/nunjucks-helper-functions.md) 
+  * [Accessing images inside related pages](tutorials/core-concepts/09-working-with-templates/thumbnails-and-areas-of-child-pages.md)
+  * [Responsive Images](tutorials/core-concepts/09-working-with-templates/responsive-images.md)
+* [Apostrophe Search](tutorials/core-concepts/10-apostrophe-search/README.md)
+  * [Enabling Search](tutorials/core-concepts/10-apostrophe-search/search.md)
 
+## Advanced Topics
+* [Advanced Pieces Topics](tutorials/advanced-development/01-advanced-pieces-topics/README.md)
+  * [AJAX Features: Enhanced Browsing for Pieces](tutorials/advanced-development/01-advanced-pieces-topics/01-ajax-pieces.md)
+  * [Fine-grained Permissions for Pieces](tutorials/advanced-development/01-advanced-pieces-topics/02-permissions-for-pieces.md)
+  * [Adding Filters](tutorials/advanced-development/01-advanced-pieces-topics/04-adding-filters.md)
+  * [Adding Columns](tutorials/advanced-development/01-advanced-pieces-topics/05-adding-columns.md)
+  * [Using the Pieces Index](tutorials/advanced-development/01-advanced-pieces-topics/06-next-previous-links.md)
+  * [Adding Fields to All Pieces](tutorials/advanced-development/01-advanced-pieces-topics/07-adding-fields-to-all-pieces.md)
+  * [Adding new batch operations for pieces](tutorials/advanced-development/01-advanced-pieces-topics/08-adding-new-batch-operations.md)
+  * [Extending the Pieces Editor Modal](tutorials/advanced-development/01-advanced-pieces-topics/09-extending-the-pieces-editor-modal.md)
+* [Advanced Pages Topics](tutorials/advanced-development/02-advanced-pages-topics/README.md)
+  * [Custom Schemas for Pages](tutorials/advanced-development/02-advanced-pages-topics/customer-schemas-for-pages.md)
+* [How Apostrophe handles requests](tutorials/advanced-development/how-apostrophe-handles-requests.md)
+* [How Apostrophe modules are structured](tutorials/advanced-development/how-apostrophe-modules-are-structured.md)
+* [Promise Events](tutorials/advanced-development/03-promise-events/promise-events.md)
```

Comment:
"Promise Events" alone doesn't really convey why you'd read it. How about "Promise Events: how to add custom server-side event handlers to Apostrophe"